branchinfo.txt

```diff
@@ -3,7 +3,7 @@
 # Each line is expected to be in the format "[Name]=[Value]".
 MAJOR_VERSION=1
 MINOR_VERSION=0
-PATCH_VERSION=0
+PATCH_VERSION=2
```

Comment:
This isn't right.  These are the CLI's version numbers.  We haven't shipped a CLI `1.0.0` yet, so it shouldn't be `1.0.2`.